src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    public static final int DATE_POSITION = 0;
+    public static final int NAME_POSITION = 1;
+    public static final int WORKING_HOURS_POSITION = 2;
+    public static final int INCOME_PER_HOUR_POSITION = 3;
+    public static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    public static final String SPACE = " ";
+    public static final String INFO_TEXT = "Report for period ";
+    public static final String DASH = " - ";
+    public static final String SEPARATOR = System.lineSeparator();
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
```

Comment:
Consider adding error handling for date parsing to handle potential `DateTimeParseException` when parsing `dateFrom`, `dateTo`, and `targetDate`. This will make the method more robust against invalid date formats.